gemfirexd/core/src/main/java/com/pivotal/gemfirexd/internal/engine/sql/execute/SnappyActivation.java

```diff
@@ -123,7 +115,7 @@ public final void checkStatementValidity() throws StandardException {
 
   public final int[] prepare() throws StandardException {
     try {
-      SnappyPrepareResultSet rs = createPreapreResultSet(0);
+      SnappyPrepareResultSet rs = createPreapreResultSet();
```

Comment:
You have removed the int as arg because there will always be 1 resultset expected? I think it was not being used only.